infrastructure/common-images/workspaces-base/backend/api/helpers/density_map/grid_creator.py

```diff
@@ -0,0 +1,145 @@
+                           'fontweight': rcParams['axes.titleweight'],
+                           'color': GRID_COLOR,
+                           'verticalalignment': 'baseline',
+                           'horizontalalignment': 'right'},
+                 loc='right')
+
+    # Sets axis color
+    ax.spines['bottom'].set_color(GRID_COLOR)
+    ax.spines['top'].set_color(GRID_COLOR)
+    ax.spines['left'].set_color(GRID_COLOR)
+    ax.spines['right'].set_color(GRID_COLOR)
+    ax.xaxis.label.set_color(GRID_COLOR)
+    ax.yaxis.label.set_color(GRID_COLOR)
+    ax.tick_params(axis='x', colors=GRID_COLOR)
+    ax.tick_params(axis='y', colors=GRID_COLOR)
+
+    for item in ([ax.title, ax.xaxis.label, ax.yaxis.label] +
+                 ax.get_xticklabels() + ax.get_yticklabels()):
+        item.set_fontsize(8)
+
+    plt.grid(False)
+    img = fig_to_img(fig)
+
+    # Corrects image alignment by adding a constant amount of padding to the image
+    shifted_image = _add_margin(img, 0, CONSTANT_RIGHT_OFFSET, CONSTANT_BOTTOM_OFFSET, 0)
+    return shifted_image
+
+
+def _get_ticks(canal_offset, img_to_res_ratio, shape, step=0.1):
+    # ticks are on image scale
+    ticks = np.round(
+        np.arange(
+            _get_tick_rounded(canal_offset * -1 - shape / 2, img_to_res_ratio),
+            _get_tick_rounded(canal_offset * -1 + shape / 2 + step, img_to_res_ratio), step),
+        2)
+    # tick labels apply the canal center offset
+    ticks_labels = []
+    for idx, tick in enumerate(ticks):
+        if idx % 5 == 0:
+            label = round((tick - canal_offset * -1) * img_to_res_ratio, 2)
+            if round(label, 1) * 10 == 0:
+                label = 0
+            ticks_labels.append(label)
+        else:
+            ticks_labels.append('')
+    return ticks, ticks_labels
+
+
+def _get_tick_rounded(original, ratio):
+    # gets tick value in the image scale so that it starts with 0 on the 2 decimal place
+    scaled = original * ratio
+    diff = round(round(scaled, 2) - round(scaled, 1), 2)
+    return original + diff / ratio
+
+
+def _set_size(w, h, ax=None) -> (float, float):
+    """ w, h: width, height in inches """
+    if not ax: ax = plt.gca()
+    l = ax.figure.subplotpars.left
+    r = ax.figure.subplotpars.right
+    t = ax.figure.subplotpars.top
+    b = ax.figure.subplotpars.bottom
+    fig_w = float(w) / (r - l)
+    fig_h = float(h) / (t - b)
+    ax.figure.set_size_inches(fig_w, fig_h)
+    return fig_w, fig_h
+
+
+def _add_margin(pil_img, top, right, bottom, left, color=(255, 0, 0, 0)) -> Image:
+    width, height = pil_img.size
+    new_width = int(width + right + left)
+    new_height = int(height + top + bottom)
+    result = Image.new(pil_img.mode, (new_width, new_height), color)
+    result.paste(pil_img, (left, top))
+    return result
+
+
+def _get_subdivision_depth(bg_atlas: ICustomAtlas, subdivision: str, resolution: int) -> Optional[int]:
+    reference = 'C8-Rostral'
```

Comment:
should we add a ToDo here? Because it's using a string for referencing

Reply:
I don't see the need for a ToDo but I think moving the reference to an attribute of the function, move 'C8-Rostral' to a setting and use that setting as the default value would be nicer than what I currently have.